recipes/legit/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kennethreitz/legit
+  git_tag: v{{ version }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - legit = legit.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - {{ packagename }}
+
+  commands:
+    - cd {{ environ.SRC_DIR }} && legit help
```

Comment:
Thanks for the help here @msarahan . 

@ericmjl try this:
```diff
diff --git a/recipes/legit/meta.yaml b/recipes/legit/meta.yaml
index 4aa0ab0..77bc633 100644
--- a/recipes/legit/meta.yaml
+++ b/recipes/legit/meta.yaml
@@ -32,7 +32,15 @@ test:
     - {{ packagename }}
 
   commands:
-    - cd {{ environ.SRC_DIR }} && legit help
+    - mkdir test-git-repo
+    - cd test-git-repo
+    - git init
+    # legit needs a remote or it exits with a non-zero status
+    # so we just need to add a fake one
+    - git remote add origin http://
+    - legit help
+  requires:
+    - git
 
 about: 
   home: https://github.com/kennethreitz/legit
```

It passes locally on my linux laptop. I cannot guarantee that this works on windows.  If it doesn't then you'll need to put the content of `commands` into a `build.sh` file and then translate the `build.sh` into a `build.bat` to make windows happy.